.gitlab-ci.yml

```diff
@@ -4,7 +4,7 @@ include:
 # SETUP
 
 variables:
-  CURRENT_CI_IMAGE: "9"
+  CURRENT_CI_IMAGE: "10"
```

Comment:
By curiosity, what does it stand for?

Reply:
It is just the tag for the Docker image. We usually bump it, so that we don't overwrite existing one. If we overwrite, there is a chance that builds from older branches won't work with a newer Docker image (because of some env incompatibility), so it is better to stay on the safer side.